src/schema/v2/me/index.ts

```diff
@@ -637,6 +641,55 @@ export const meType = new GraphQLObjectType<any, ResolverContext>({
         return !!share_follows
       },
     },
+    // genomic recommendation
+    recommendedArtworks: {
```

Comment:
There's already a root-level `artworksForUser` field that returns recommendations. Now there's a `recommendedArtworks` under `me` that returns _different_ recommendations. I think we should consolidate the schema a bit!

Reply:
Yeah! I was thinking similar, but this is what I thought would be the "less" ambiguous option.

We have the `artworkRecommendations` under Me, and I believe `artworksForUser` is the same.. Should we use the `artworksForUser` here instead?

Reply:
It seems like I can recollect more activity happening recently in `artworksForUser` (Braze usage and such) - I wasn't aware of the `artworkRecommendations`.

I _think_ we should deprecate `artworkRecommendations`, and add this to `artworksForUser` as that has all the functionality (auction lot recs, etc.).

Possibly we can add a `mode: AFFINITY|GENOMIC` or something like that to allow that same field to be used?

(also, minor but it'd be great if that field was called `artworkRecommendationsForUser` now that it seems to be the different kinds of recs. that are being exposed)

What do you think?